Pages/fashionCategoryPage.js

```diff
@@ -0,0 +1,31 @@
+class FashionCategoryPage{
+    constructor(){
+        this.title ="Fashion Category"
+    }
+    get brownJacket(){
+        return browser.actions().mouseMove(element(by.xpath("//*[@id='root']/div/div[1]/div[3]/div/div[1]/div/div/a/img")));
+    }
+    get blueJacket(){
+        return browser.actions().mouseMove(element(by.xpath(" //*[@id='root']/div/div[1]/div[3]/div/div[3]/div/div/a/img")));
```

Comment:
Xpath is not a good code pratice (read about it here: https://www.protractortest.org/#/style-guide), can you consider using some other options where it is possible ?

Reply:
Where I could add better locator, I added it, there are few places where I had to leave it, so that I don't use loops in methods